### PR TITLE
Anf 10997 NetApp Files update swagger rest api to 2021-02-01

### PR DESCRIPTION
**Pull request by audunn**
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i>

### Changelog
Please ensure to add changelog with this PR by answering the following questions.
  1. What's the purpose of the update?    
      - [ ] new service onboarding 
      - [x] new API version 
      - [ ] update existing version for new feature 
      - [ ] update existing version to fix swagger quality issue in s360
      - [ ] Other, please clarify 
  2. When you are targeting to deploy new service/feature to public regions? Please provide date, or month to public if date is not available yet.
  3. When you expect to publish swagger? Please provide date, or month to public if date is not available yet.
  4. If it's an update to existing version,  please select SDKs of specific language and CLIs that require refresh after swagger is published.
      - [ ] SDK of .NET (need service team to ensure code readiness)
      - [ ] SDK of Python
      - [ ] SDK of Java
      - [ ] SDK of Js
      - [ ] SDK of Go
      - [ ] PowerShell
      - [ ] CLI
      - [ ] Terraform
      - [ ] No, no need to refresh for updates in this PR

### Contribution checklist:
- [x] I commit to follow the [Breaking Change Policy](http://aka.ms/bcforapi) of "no breaking changes"
- [x] I have reviewed the [documentation](https://aka.ms/ameonboard) for the workflow.
- [x] [Validation tools](https://aka.ms/swaggertools) were run on swagger spec(s) and errors have all been fixed in this PR. [How to fix?](https://aka.ms/ci-fix)

If any further question about AME onboarding or validation tools, please view the [FAQ](https://aka.ms/faqinprreview).

### ARM API Review Checklist
- [x] Ensure to check this box if one of the following scenarios meet updates in the PR, so that label “WaitForARMFeedback” will be added automatically to involve ARM API Review. Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs, all “removals” and “adding a new property” no more require ARM API review.
  - Adding new API(s)
  - Adding a new API version
  -  [x] Ensure to copy the existing version into new directory structure for first commit (including refactoring) and then push new changes including version updates in separate commits. This is required to review the changes efficiently.
  - Adding a new service

- [x] Please ensure you've reviewed following [guidelines](https://aka.ms/rpguidelines) including [ARM resource provider contract](https://github.com/Azure/azure-resource-manager-rpc) and [REST guidelines](https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md). Estimated time (4 hours). This is required before you can request review from ARM API Review board.

- [ ] If you are blocked on ARM review and want to get the PR merged with urgency, please get the ARM oncall for reviews (*RP Manifest Approvers* team under <ins>Azure Resource Manager service</ins>) from IcM and reach out to them. 

### Breaking Change Review Checklist 
If there are following updates in the PR, ensure to request an approval from Breaking Change Review Board as defined in the [Breaking Change Policy](http://aka.ms/bcforapi). 

- [ ] Removing API(s) in stable version
- [ ] Removing properties in stable version
- [ ] Removing API version(s) in stable version
- [ ] Updating API in stable or public preview version with Breaking Change Validation errors
- [ ] Updating API(s) in public preview over 1 year (refer to [Retirement of Previews](https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/37683/Retirement-of-Previews))

**Action**: to initiate an evaluation of the breaking change, create a new intake using the [template for breaking changes](https://aka.ms/Breakingchangetemplate). Addition details on the process and office hours are on the [Breaking change Wiki](https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/37684/Breaking-Changes).

Please follow the link to find more details on [PR review process](https://aka.ms/SwaggerPRReview).

### CODE REVIEW

File: specification/netapp/resource-manager/Microsoft.NetApp/stable/2021-02-01/examples/Accounts_Get.json
Comment (zcxcasd) [inline specification/netapp/resource-manager/Microsoft.NetApp/stable/2021-02-01/examples/Accounts_Get.json:32]: perhaps add example for     "accountEncryption/ keyVaultProperties that you added?

**Comment by audunn** [comment]
Note VolumeBackupStatus_Get  had to be renamed to Backups_GetStatus on request form Azure CLI team.   They demand it be change as it cannot be grouped with backup operations in the CLI  due the the operation id. 
Note also that this operation was added during last update (2020-12-01) and backup is a preview feature.

File: specification/netapp/resource-manager/Microsoft.NetApp/stable/2021-02-01/examples/Accounts_Get.json
Comment (audunn) [reply specification/netapp/resource-manager/Microsoft.NetApp/stable/2021-02-01/examples/Accounts_Get.json:46]: Aded

**Review by hildurhodd — APPROVED** [review]

File: specification/netapp/resource-manager/Microsoft.NetApp/stable/2021-02-01/netapp.json
Comment (audunn) [reply specification/netapp/resource-manager/Microsoft.NetApp/stable/2021-02-01/netapp.json]: Already done
`        "operationId": "Backups_GetStatus",`

**Review by majastrz — APPROVED** [review]
Signed off from ARM side.

File: specification/netapp/resource-manager/readme.java.md
Comment (akning-ms) [inline specification/netapp/resource-manager/readme.java.md:52]: @weidongxu-microsoft to confirm the change for Java readme

Comment (weidongxu-microsoft) [reply specification/netapp/resource-manager/readme.java.md:52]: It is fine, though not necessary. For T2 Java SDK build with swagger will follow the default tag in readme.md. In release request you can specify the tag, though we might not release old tag if current Java SDK is on new one.

Comment (akning-ms) [reply specification/netapp/resource-manager/readme.java.md:52]: Thanks Weidong,  hi @audunn, let me know if you need to merge this PR

File: specification/netapp/resource-manager/Microsoft.NetApp/stable/2021-02-01/netapp.json
Comment (akning-ms) [reply specification/netapp/resource-manager/Microsoft.NetApp/stable/2021-02-01/netapp.json]: FYI, it may cause SDK breaking change if change operationId.  pls confirm it is what you expected

Comment (akning-ms) [reply specification/netapp/resource-manager/Microsoft.NetApp/stable/2021-02-01/netapp.json]: saw the comments, pls ignore it

File: specification/netapp/resource-manager/readme.java.md
Comment (audunn) [reply specification/netapp/resource-manager/readme.java.md:52]: > Thanks Weidong, hi @audunn, let me know if you need to merge this PR

Perhaps best to hold off for now as I hear deployments of this api-version are being rollined out and the api-version is yet not available in all locations.  Should be soon I hope and I will update here once they are out.

**Comment by audunn** [comment]
Note needed to add snapshotPolicyId to volumepatch properties as that omission is causing issues for customers assigning snapshotpolicy to existing volumes.

**Comment by audunn** [comment]
@akning-ms We can move ahead with this now

**Comment by akning-ms** [comment]
Pls remove specification/netapp/resource-manager/readme.azureresourceschema.md from your PR which is not need any more and caused conflict

**Comment by audunn** [comment]
@akning-ms removed
> Pls remove specification/netapp/resource-manager/readme.azureresourceschema.md from your PR which is not need any more and caused conflict

**Comment by akning-ms** [comment]
Hi audunn, 

Looks you removed a property which is a breaking change in recent change. it require breaking change review board approval
message: "The new version is missing a property found in the old version. Was 'tags' renamed or removed?"
Can you explain the reason?

**Comment by akning-ms** [comment]
> > Hi audunn,
> > Looks you removed a property which is a breaking change in recent change. it require breaking change review board approval
> > message: "The new version is missing a property found in the old version. Was 'tags' renamed or removed?"
> > Can you explain the reason?
> 
> Yes, tags was defined in on backupPatch. Backup is a proxy resource so we thought is should go. It is not in the backup create properties and it does nothing, updating tags does not do anything,
> 
> Should we add it back to expedite this or keep it removed?

Yes. to speed up PR, you should add it back, otherwise you should get breaking change review board approval before I can help merge the PR.

**Comment by audunn** [comment]
@akning-ms tags added back to backupPatch. To be done in a later version.

**Pull request merged**
